src/tests/subscribe.rs

```diff
@@ -0,0 +1,105 @@
+use super::mock::*;
+use crate::Error;
+use frame_support::{assert_noop, assert_ok};
+
+#[test]
+fn subscribe() {
+	ExternalityBuilder::build().execute_with(|| {
+		assert_ok!(PalletSubscription::subscribe(
+			Origin::signed(1),
+			2,
```

Comment:
Define your users as constants in the mock file.
const ALICE = 1
const BOB = 2

Reply:
Done !